src/Login/components/LoginCallback/LoginCallback.tsx

```diff
@@ -1,38 +1,15 @@
-import { useEffect } from 'react';
-import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
 
-import { getLoginToken } from '../../api/getLoginToken';
-import { postLoginToken } from '../../api/postLoginToken';
+import useGetLoginToken from '../../hooks/useGetLoginToken';
+import usePostLoginToken from '../../hooks/usePostLoginToken';
 
 function LoginCallback() {
-  const navigate = useNavigate();
+  const [loginToken, setLoginToken] = useState('');
 
-  useEffect(() => {
-    const fetchData = async () => {
-      try {
-        const tokenRes = await getLoginToken();
+  const getMutation = useGetLoginToken({ loginToken, setLoginToken });
```

Comment:
P2: set 함수 그대로 내리기 보단 요런 식으로 핸들링 함수 하나 만들어서 내려주는건 어떨까요!?
이번에 리팩토링을 하면서 set 함수 -> 핸들링 함수로 바꾼 뒤, props로 내려줘야겠다고 생각했어용. 유지보수 시에 컴포넌트 전반적인 코드 수정 없이 함수 내부 코드만 변경해주면 돼서 훨씬 간편하더라구요!
추후 리팩토링까지 생각한다면, 아래처럼 수정해봐도 좋을 것 같습니당 ~

```suggestion
  const handleLoginToken = (token) => {
    setLoginToken(token)
  }

  const getMutation = useGetLoginToken({ loginToken, handleLoginToken });
```

Reply:
P2: 아름이가 위에 잘 적어줬네요~!! setter 함수를 그대로 내려주는 건 추상화나 책임분리 측면에서도 좋지 못한 패턴이라구 생각해요!!!

Reply:
헉 리팩토링 하면서 setter 함수를 그대로 내렸던 것들을 핸들링 함수로 바꾸려고 했었는데 까먹었네요ㅜㅜ 제가 맡았던 부분들 다 바꿔서 반영해두도록 할게요! 쩡우랑 루밍이 알려줘서 너무너무 고마워요!!